kong/conf_loader/init.lua

```diff
@@ -137,6 +137,11 @@ local cipher_suites = {
 }
 
 
+-- using kong version, "major.minor"
+local LMDB_VALIDATION_TAG = string.match(require("kong.meta").version,
```

Comment:
Let's not parse the version number like this, we can simply do a `string.format("%d.%d", meta._VERSION_TABLE.major, meta._VERSION_TABLE.minor)` which is much more straightforward.

Reply:
done.